website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,226 @@
+
+    ```shell
+
+    cd jaffle_shop_duck_db
+
+    ```
+
+
+3. Install dbt Core and DuckDB in a virtual environment.
+
+    <Expandable alt_header="Example for Mac" >
+
+    ```shell
+
+    python3 -m venv venv
+    source venv/bin/activate
+    python3 -m pip install --upgrade pip
+    python3 -m pip install -r requirements.txt
+    source venv/bin/activate
+
+    ```
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows" >
+
+    ```shell
+
+    python -m venv venv
+    venv\Scripts\activate.bat
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\activate.bat
+
+    ```
+
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows PowerShell" >
+
+    ```shell
+
+    python -m venv venv
+    venv\Scripts\Activate.ps1
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\Activate.ps1
+
+    ```
+    </Expandable>
+
+
+4. Ensure your profile is setup correctly from the command line by running the following:
+
+
+    - [dbt compile](https://docs.getdbt.com/reference/commands/compile) &mdash; generates executable SQL from your project source files
+    - [dbt run](https://docs.getdbt.com/reference/commands/run) &mdash; compiles and runs your project
+    - [dbt test](https://docs.getdbt.com/reference/commands/test) &mdash; compiles and tests your project
+    - [dbt build](https://docs.getdbt.com/reference/commands/build) &mdash; compiles, runs, and tests your project
+    - [dbt docs generate](/reference/commands/cmd-docs#dbt-docs-generate) &mdash; generates your project's documentation.
+    - [dbt docs serve](/reference/commands/cmd-docs#dbt-docs-serve) &mdash; starts a webserver on port 8080 to serve your documentation locally and opens the documentation site in your default browser.
+
+    For complete details, refer to the [dbt command reference](/reference/dbt-commands).
+
+:::note
+
+The steps will fail if you decide to run this project in your data warehouse (outside of this DuckDB demo). You will need to reconfigure the project files for your warehouse. Definitely consider this if you are using a community-contributed adapter.
+
+:::
+
+
+### Troubleshoot
+
+
+
+   <Expandable alt_header="Could not set lock on file error" >
+
+    ```Jinja
+
+    IO Error: Could not set lock on file "jaffle_shop.duckdb": Resource temporarily unavailable
+
+    ```
+
+    This is a known issue in DuckDB. Try disconnecting from any sessions that are locking the database. If you are using DBeaver, this means shutting down DBeaver (disconnecting doesn't always work).
+
+    As a last resort, deleting the database file will get you back in action (_but_ you will lose all your data).
+
+   </Expandable>
+
+
+</TabItem>
+ 
+<TabItem value="web" label="Web browser">
+
+1. Go to the `jaffle-shop-template` [repository](https://github.com/dbt-labs/jaffle-shop-template) after you log in to your GitHub account. 
```

Comment:
is this the right template we want to share? it looks like its archived?  should [this be used](https://github.com/gwenwindflower/octocatalog) instead?
![Screenshot 2024-10-14 at 15 54 38](https://github.com/user-attachments/assets/313ecc79-f27d-4454-889e-41373313c768)

Reply:
if the link is incorrect, then do the steps in the 'web browser' tab need changing?

Reply:
hey @nataliefiann , i noticed you changed the link but i would recommend triple checking this with the sme